tests/Integration/Controllers/OccurrenceSpeciesController/IndexEndpointTest.php

```diff
@@ -0,0 +1,179 @@
+{
+    use RefreshDatabase;
+
+    /**
+     * @test
+     */
+    public function itFailsToIndexOccurrenceSpeciesDueToInvalidContentTypeHeader(): void
+    {
+        $response = $this->json('GET', route('occurrences::species::index'));
+
+        $response->assertHeader('Content-Type', 'application/vnd.api+json');
+        $response->assertStatus(415);
+        $response->assertJson([
+            'errors' => [
+                [
+                    'status' => 415,
+                    'detail' => 'Wrong media type',
+                ],
+            ],
+        ]);
+    }
+
+    /**
+     * @test
+     */
+    public function itFailsToIndexOccurrenceSpeciesDueToInvalidInput(): void
+    {
+        $response = $this->json('GET', route('occurrences::species::index'), [
+            'page' => [
+                'number' => 'second',
+                'size'   => 'ten',
+            ],
+            'search'   => '',
+            'exact'    => 'yes',
+            'families' => [
+                1,
+            ],
+            'sort'  => 'id',
+            'order' => 'up',
+        ], [
+            'Content-Type' => 'application/vnd.api+json',
+        ]);
+
+        $response->assertHeader('Content-Type', 'application/vnd.api+json');
+        $response->assertStatus(422);
+        $response->assertJson([
+            'errors' => [
+                [
+                    'detail' => 'The page.number must be an integer.',
+                    'meta'   => [
+                        'field' => 'page.number',
+                    ],
+                ],
+                [
+                    'detail' => 'The page.size must be an integer.',
+                    'meta'   => [
+                        'field' => 'page.size',
+                    ],
+                ],
+                [
+                    'detail' => 'The search must be a string.',
+                    'meta'   => [
+                        'field' => 'search',
+                    ],
+                ],
+                [
+                    'detail' => 'The exact field must be true or false.',
+                    'meta'   => [
+                        'field' => 'exact',
+                    ],
+                ],
+                [
+                    'detail' => 'The selected sort is invalid.',
+                    'meta'   => [
+                        'field' => 'sort',
+                    ],
+                ],
+                [
+                    'detail' => 'The selected order is invalid.',
+                    'meta'   => [
+                        'field' => 'order',
+                    ],
+                ],
+                [
+                    'detail' => 'The selected families.0 is invalid.',
+                    'meta'   => [
+                        'field' => 'families.0',
+                    ],
+                ],
+            ],
+        ]);
+    }
+
+    /**
+     * @test
+     */
+    public function itSuccessfullyIndexeSpeciesOccurrences(): void
```

Comment:
`Indexe` typo?

Reply:
Thanks, I saw this one when looking at the test output. Already sorted.